### PR TITLE
 Deleting old specs and adding one for the company account spec [#129] 

### PR DESCRIPTION
**Pull request by Mstc08**
Adding a new spec for company account javascript file

### CODE REVIEW

**Comment by kjanei** [comment]
Please clarify commit messages in the future:
Commit 8a0fd20: Removed question-promptSpec.js because its corresponding javascript file is not used [#129]
Commit 2cab1ff: Removed questionFrontEndSpec.js because its corresponding javascript file is not used [#129]
Commit 81b0ba9: Removed questionsSpec.js because its corresponding javascript file is not used [#129]
Commit 27f25ec: Removed userSpec.js because its corresponding javascript file is not used [#129]
Commit 7172dc0: Add tests for the company account javascript file [#129]

File: test/companyAccountSpec.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+});
+
+const testCompanyAccount = mongoose.model('testCompany', testCompanySchema);
+describe('Connecting to database for company account', function() {
+  before(function (done) {
+    mongoose.connect('mongodb://soen341:soen341@soen341-shard-00-00-ruxjj.mongodb.net:27017,soen341-shard-00-01-ruxjj.mongodb.net:27017,soen341-shard-00-02-ruxjj.mongodb.net:27017/test?ssl=true&replicaSet=SOEN341-shard-0&authSource=admin');
+    const db = mongoose.connection;
+    db.on('error', console.error.bind(console, 'connection error'));
+    db.once('open', function() {
+      console.log('We are connected to test database for company account!');
```
Comment (kjanei) [inline test/companyAccountSpec.js]: Please remove unnecessary console.log

File: test/companyAccountSpec.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+      done();
```
Comment (kjanei) [inline test/companyAccountSpec.js]: Please keep all indentation to four spaces. (This applies to the rest of the file as well.)

File: test/companyAccountSpec.js (hunk 1 above)
Comment (Mstc08) [reply test/companyAccountSpec.js]: We have to keep the console log to know we are actually connected to the database and keep track of the sequence of tests

**Pull request merged**